### PR TITLE
feat: As a user, I want to be able to see the issuer details page

### PR DESCRIPTION
**Pull request by OliynykPro**
## What does this PR do?
Issuer page has been added

### Related ticket
https://github.com/Consensys/linea-attestation-registry/issues/417

### CODE REVIEW

File: explorer/src/pages/Home/interface.ts
Comment (ars9) [inline explorer/src/pages/Home/interface.ts:1]: My apologies, I haven't updated the payload format. It should look like this:

```json
...
{
    "name": "Trusta",
    "logoURL": "trusta-white.jpg",
    "keywords": "reputation, wallet score",
    "CTATitle": "Learn more",
    "description": "Connect your wallet to LINEA to view and generate your Trusta's Sybil Score attestation. Your humanity is verified based only on your on-chain data, without needing approval from a central authority, and without any request for your personal info.",
    "CTALink": "https://trustgo.trustalabs.ai/etrusta/0x085ed975a8b6b860de3c2b871da60a3f9f48a5b8/lineaverax/h?f=linea&chainId=324"
}
...
```

File: explorer/src/pages/Issuer/components/Banner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <img src={issuerBG} alt="issuer background" className="absolute -z-[1] top-0 left-0 h-full w-full object-cover" />
+      <div className="mb-6 lg:mb-0 md:flex md:items-center md:gap-6">
+        <IssuerLogo className="mb-4 md:mb-0 w-[2.875rem] md:w-16 h-auto" />
+        <div className="text-[2rem] font-medium md:text-[2.5rem] lg:text-[2.75rem] md:font-semibold text-whiteDefault">
+          {name}
+        </div>
+      </div>
+      {landingPage && (
+        <Button
+          name={t("common.actions.learnMore")}
```
Comment (ars9) [inline explorer/src/pages/Issuer/components/Banner/index.tsx:24]: Button text should be pulled from "CTATitle" field (please see the comment above).

File: explorer/src/pages/Issuer/components/Description/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { IDescriptionProps } from "./interface";
+
+export const Description: React.FC<IDescriptionProps> = ({ description }) => {
+  return (
+    <div className="flex flex-col gap-6 lg:gap-8 md:w-[31.6875rem] lg:w-[40.3125rem] max-w-full m-auto">
+      {/* TODO: uncomment when data will be available */}
```
Comment (ars9) [inline explorer/src/pages/Issuer/components/Description/index.tsx:6]: Please take this list from `keywords` field from Issuers payload.

**Review by ars9 — CHANGES_REQUESTED** [review]

File: explorer/src/pages/Issuer/components/Portals/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { Button } from "@/components/Buttons";
+import { EButtonType } from "@/components/Buttons/enum";
+import { HelperIndicator } from "@/components/HelperIndicator";
+
+// TODO: add dynamic data via props
+export const Portals: React.FC = () => {
+  return (
+    <div className="flex flex-col gap-6 md:gap-8">
+      <div className="flex items-center text-2xl md:text-[2rem] font-semibold gap-2">
+        <HelperIndicator type="portal" sizeClass="w-1 h-4 md:w-[0.375rem] h-6" /> Portals
```
Comment (voenkomatiwe) [inline explorer/src/pages/Issuer/components/Portals/index.tsx:13]: h-4 or h-6. And I would recommend doing this
`size="big"` and inside make it `type Size = "small" | "big"`

**Review by ars9 — APPROVED** [review]

**Review by Solniechniy — APPROVED** [review]

**Pull request merged**